test/functional/services/apps_menu.js

```diff
@@ -26,15 +26,16 @@ export function AppsMenuProvider({ getService }) {
   return new class AppsMenu {
     async readLinks() {
       await this._ensureMenuOpen();
-      const buttons = await testSubjects.findAll('appsMenu appLink');
-      try {
-        return Promise.all(buttons.map(async (element) => ({
-          text: await element.getVisibleText(),
-          href: await element.getProperty('href'),
-        })));
-      } finally {
-        await this._ensureMenuClosed();
-      }
+      const appMenu = await testSubjects.find('navDrawer&expanded appsMenu');
+      const $ = await appMenu.parseDomContent();
+      const links = $('[data-test-subj="appLink"]').toArray().map(link => {
```

Comment:
Did we not find a way to extend Cheerio with testSubject methods? I would bet that $.fn is an option if Cheerio really wants to expose the jQuery API

Reply:
@spalger I was working on POC for that and I have several questions to clarify about shimming. After discussion with @LeeDr I postponed it as we agreed it is lower prio than WD migration and  can be done in separate PR later.